server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GetAssertionsTests.java

```diff
@@ -0,0 +1,248 @@
+of the GNU Affero General Public License as published by the Free Software Foundation, either
+version 3 of the License, or (at your option) any later version.
+
+The colorado-rla IRV extensions are distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+See the GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along with
+raire-service. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+package au.org.democracydevelopers.corla.endpoint;
+
+import au.org.democracydevelopers.corla.model.ContestType;
+import au.org.democracydevelopers.corla.util.testUtils;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import us.freeandfair.corla.model.*;
+
+import java.io.*;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Set;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+import java.util.zip.ZipInputStream;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Test the GetAssertions endpoint, both CSV and JSON versions. The response is supposed to be a zip file containing
+ * the assertions.
+ * TODO This really isn't a completely comprehensive set of tests yet. We also need:
+ * - API testing
+ * - Testing for retrieving the data from the zip.
+ * - Testing that the service throws appropriate exceptions if the raire service connection isn't set up properly.
+ * See <a href="https://github.com/DemocracyDevelopers/colorado-rla/issues/125">...</a>
+ */
+public class GetAssertionsTests {
+
+    private static final Logger LOGGER = LogManager.getLogger(GetAssertionsTests.class);
+
+    private final static String boulderMayoral = "City of Boulder Mayoral Candidates";
+    private final static String tinyIRV = "TinyExample1";
+
+    private final Choice alice = new Choice("Alice", "", false, false);
+    private final Choice bob = new Choice("Bob", "", false, false);
+    private final Choice chuan = new Choice("Chuan", "", false, false);
+
+    private final List<Choice> boulderMayoralCandidates = List.of(
+            new Choice("Aaron Brockett", "", false, false),
+            new Choice("Nicole Speer", "", false, false),
+            new Choice("Bob Yates", "", false, false),
+            new Choice("Paul Tweedlie", "", false, false)
+    );
```

Comment:
Note that the following javadoc comments will only be associated with the method directly under them, not for any of the subsequent ones.